src/.nuget/Microsoft.NETCore.Jit/win/Microsoft.NETCore.Jit.pkgproj

```diff
@@ -9,9 +9,13 @@
   </PropertyGroup>
   <ItemGroup>
     <ArchitectureSpecificNativeFile Include="$(BinDir)clrjit.dll" />
+    <CrossArchitectureSpecificNativeFile Include="$(BinDir)$(CrossTargetComponentFolder)\clrjit.dll" />
     <File Include="@(ArchitectureSpecificNativeFile)">
       <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
     </File>
+    <File Condition="'$(HasCrossTargetComponents)' == 'true'" Include="@(CrossArchitectureSpecificNativeFile)">
+      <TargetPath>runtimes/$(CrossTargetComponentFolder)_$(PackagePlatform)/native</TargetPath>
```

Comment:
Even native crossgen (for any arch) under `tools` requires native clrjit under `runtimes`. May be `runtimes` folder name itself is not right one regardless of native or cross-target clrjit.  Let's wait what @gkhanna79 says.